Cargo.toml

```diff
@@ -17,6 +17,8 @@ RustBootServicesAllocatorDxe = {path = "MsCorePkg/Crates/RustBootServicesAllocat
 HidIo = {path = "HidPkg/Crates/HidIo"}
 hidparser = {git = "https://github.com/microsoft/mu_rust_hid.git", branch = "main"}
 HiiKeyboardLayout = {path = "HidPkg/Crates/HiiKeyboardLayout"}
+mu_rust_helpers = {git = "https://github.com/microsoft/mu_rust_helpers.git", branch = "main"}
+boot_services = { git = "https://github.com/microsoft/mu_rust_helpers.git", branch = "main" }

```

Comment:
I'm leaning toward using tagged versions here to prevent break on latest. Any concern with that?

Reply:
Thanks for updating to use the tagged release